pkg/cli/cmd/env/show/preview/show.go

```diff
@@ -176,6 +176,12 @@ func (r *Runner) Run(ctx context.Context) error {
 		if a.RecipePack > b.RecipePack {
 			return 1
 		}
+		if a.ResourceType < b.ResourceType {
+			return -1
+		}
+		if a.ResourceType > b.ResourceType {
+			return 1
+		}
 		return 0
 	})
```

Comment:
```suggestion
slices.SortFunc(envRecipes, func(a, b EnvRecipes) int {
    if v := cmp.Compare(a.RecipePack, b.RecipePack); v != 0 {
        return v
    }
    return cmp.Compare(a.ResourceType, b.ResourceType)
})
```

For clearer ordering and reduces boilerplate.